### ISSUE TITLE
MonacoEditor as a feature module

### ISSUE THREAD

**Issue by dasco144** [alauda/ng-monaco-editor#61]
Hi

I am trying to reduce my initial bundle size and one thing I noticed is the the MonacoEditorModule is importing a few libraries/modules (FormsModule, ReactiveFormsModule, lodash-es) that could possibly be lazy loaded at a later stage as I am only using the editor in a module that is lazy loaded for a specific route.

I have tried moving the forRoot import into the feature module but I get the following error:
```
ERROR NullInjectorError: R3InjectorError(AppModule)[MonacoEditorConfig -> MonacoEditorConfig -> MonacoEditorConfig]: 
  NullInjectorError: No provider for MonacoEditorConfig!
```

After seeing this I tried to provide this config myself but I was unable to get it working.

Is there a way to get the MonacoEditorModule as an import in a feature module and not in the app module?

**Comment by JounQin** [comment]
You'd better provide a runnable production.

---

You can try to just provide `MonacoEditorConfig` in the root module.

**Comment by dasco144** [comment]
I've tried to recreate the issue in a new repo, but I can't seem to replicate the issue. I have that repo here https://github.com/dasco144/ng-monaco-test

On my main project (which I unfortunately can't share much code from) I tried providing `MonacoEditorConfig` and `MonacoProviderService` in the app module, but this seems to pull in the same modules (FormsModule, ReactiveFormsModule, lodash-es) as the `MonacoEditorModule` import.

I think the way the module is being loaded for my feature module is perhaps causing the issue. I have the component loading via a primeng dialog service.

I'll have to investigate a bit more
